packages/plugin-core/src/web/extension.ts

```diff
@@ -1,26 +1,25 @@
+import { TreeViewItemLabelTypeEnum } from "@dendronhq/common-all";
 import "reflect-metadata"; // This needs to be the topmost import for tsyringe to work
 
 import { container } from "tsyringe";
 import * as vscode from "vscode";
 import { NoteLookupCmd } from "./commands/NoteLookupCmd";
 import { setupWebExtContainer } from "./injection-providers/setupWebExtContainer";
+import { NativeTreeView } from "./views/treeView/NativeTreeView";
 
 /**
  * This is the entry point for the web extension variant of Dendron
  * @param context
  */
 export async function activate(context: vscode.ExtensionContext) {
   // Use the web extension injection container:
-  try {
```

Comment:
did we decide to get rid of this try/catch?

Reply:
No - this is a merge error, thanks for the catch.